src/openrct2/object/ObjectManager.cpp

```diff
@@ -232,6 +233,21 @@ class ObjectManager final : public IObjectManager
         return objects;
     }
 
+    static rct_string_id GetObjectSourceGameString(const rct_object_entry * entry)
+    {
+        uint8 source = (entry->flags & 0xF0) >> 4;
+        switch (source) {
+            case OBJECT_SOURCE_RCT2:
```

Comment:
Sorry - it seems to be indented in lots of (most?) other places, so I just followed that! Noted for future reference.